setup.py

```diff
@@ -27,6 +27,7 @@
     long_description=long_description,
     license="MIT",
     packages=["pyscreeze"],
+    package_data={"pyscreeze": ["pyscreeze/py.typed"]},
```

Comment:
Alternatively we can add a lower-bound pin on setuptools to https://setuptools.pypa.io/en/latest/history.html#v69-0-0 , where `py.typed` files are picked-up by default.